pkg/pot/lexer.go

```diff
@@ -0,0 +1,132 @@
+package pot
+
+import (
+	"bufio"
+	"strconv"
+	"strings"
+)
+
+type TokenType int
+
+const (
+	HeaderLanguage TokenType = iota
+	HeaderTranslator
+	HeaderPluralForms
+	MsgCtxt
+	MsgId
+	MsgStr
+	PluralMsgId
+	PluralMsgStr
+	TranslatorComment
+	ExtractedComment
+	Reference
+	Flag
+	MsgctxtPreviousContext
+	MsgidPluralPrevUntStrPlural
+	MsgidPrevUntdStr
+)
+
+type Token struct {
+	Value string
+	Type  TokenType
+	Index int // plural index for msgstr with plural forms
+}
+
+func Lex(r *bufio.Reader) []Token {
```

Comment:
Maybe I don't understand but this function is used only in tests, why is it needed?